### PR TITLE
Object Store Changes: 1) Rename OjbecStore to LibcloudObjectStore

### PR DESCRIPTION
**Pull request by anisehsani**
Refactoring ObjectStore to eventually be an abstract class that gets implemented by both S3ObjectStore and LibcloudObjectStore.

### CODE REVIEW

File: composer/utils/object_store.py [hunk 1]
```diff
@@ -155,22 +155,22 @@ def initialize_object(self):
             ObjectStore: The object_store.
         """
 
-        return ObjectStore(
+        return LibcloudObjectStore(
```
Comment (ravi-mosaicml) [inline composer/utils/object_store.py:158]: nit: Can you also rename the class to `LibcloudObjectStoreHparams`?

**Review by ravi-mosaicml — APPROVED** [review]
LGTM; see comment.

**Comment by anisehsani** [comment]
@ravi-mosaicml for now, I'm just leaving the `ObjectStoreHparams` as is (just creating a `LibcloudObjectStore` on `initialize_object`) and will adjust the naming later. I figure they might disappear soon anyway.

**Comment by anisehsani** [comment]
ok - will rename

**Pull request merged**